src/components/BaseInfo/BaseInfo.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export function BaseInfo({ title, imageURL }) {
+  return (
+    <div>
+      <div>
+        <h2>{title}</h2>
+      </div>
+      <div>
+        {imageURL ? <img src={imageURL}></img> : ''}
```

Comment:
Better to use self-closing tag in cases with no content

`<img src={imageURL}/>` instead of `<img src={imageURL}></img>`